### PR TITLE
add some minimum sizies for some windows

### PR DESCRIPTION
**Pull request by BaraMGB**
- adds a minimum height for **controller rack window**
- adds minimum size for **about dialog** and **export dialog** (setup dialog)

### CODE REVIEW

**Comment by zonkmachine** [comment]
Yes. Those are good sizes. Maybe the size of at least the **export dialog** should be fixed as the fields looks a bit odd when you stretch it.

**Comment by BaraMGB** [comment]
bildschirmfoto vom 2017-02-07 14-51-53

I fear on different languages it look different.

**Comment by zonkmachine** [comment]
> I fear on different languages it look different.

Right. Forgot about those.

File: src/gui/dialogs/about_dialog.ui [hunk 1]
```diff
@@ -10,6 +10,12 @@
     <height>357</height>
    </rect>
   </property>
+  <property name="minimumSize">
+   <size>
+    <width>558</width>
```
Comment (Umcaruje) [inline src/gui/dialogs/about_dialog.ui]: the numbers seem odd to me, why not go with something like `560x360`?

File: src/gui/dialogs/export_project.ui [hunk 2]
```diff
@@ -10,6 +10,12 @@
     <height>412</height>
    </rect>
   </property>
+  <property name="minimumSize">
+   <size>
+    <width>519</width>
```
Comment (Umcaruje) [inline src/gui/dialogs/export_project.ui]: same here, why not `520x410`

Comment (tresf) [inline src/gui/dialogs/export_project.ui]: To me it would be more odd (statistical anomaly) that a round number were coincidentally the right number.

Comment (BaraMGB) [inline src/gui/dialogs/export_project.ui]: I took the numbers which are used as initial sizes for that windows.

Comment (Umcaruje) [inline src/gui/dialogs/export_project.ui]: Ok, no need to change them then, it's just that I found them odd at a first glance ;)

**Pull request merged**